Sources/SourceKitBazelBSP/RequestHandlers/BuildTargets/BazelTargetQuerier.swift

```diff
@@ -59,7 +60,12 @@ final class BazelTargetQuerier {
         self.commandRunner = commandRunner
     }
 
-    func queryTargets(forConfig config: BaseServerConfig, rootUri: String, kinds: Set<String>) throws -> XMLElement {
+    /// Based on the server config, it constructs query and calls `bazel query deps` with output proto
+    func queryTargets(
```

Comment:
```suggestion
    func queryTargets(
```

Ok to not have the comment in this case